src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Areas/Identity/RevalidatingAuthenticationStateProvider.cs

```diff
@@ -0,0 +1,98 @@
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
What sort order do the other template files use for these? We use "System first" for our code which is not the VS default.

Reply:
Good point. It is "system first" elsewhere in the template. Fixed this.